masterlist.yaml

```diff
@@ -2501,16 +2523,49 @@ plugins:
   - name: 'AOS.esp'
     after:
       - 'WetandCold.esp'
+      # RWT must be loaded after Requiem but AOS shouldn't so this rule is disabled when Requiem is installed
       - name: 'RealisticWaterTwo.esp'
-        condition: 'not file("AOS2_RealisticWaterTwo Patch.esp") and not file("WAO-.*RWT.*\.esp")'
+        condition: 'not file("AOS2_RealisticWaterTwo Patch.esp") and not file("WAO-.*RWT.*\.esp") and not file("Requiem.esp")'
+    msg:
+      - <<: *reqRequiemPatch
+        condition: 'file("Requiem.esp") and not active("Requiem - Audio Overhaul for Skyrim.esp") and not active("AOS2_Requiem Patch.esp")'
     dirty:
       - crc: 0xd88afe72
         <<: *dirtyPlugin
         itm: 7
         udr: 0
+  - name: 'AOS2_DD Patch.esp'
+    msg:
+      - <<: *redundantPluginX
```

Comment:
Same reason as in [#136 (comment)](https://github.com/loot/skyrim/pull/136#discussion_r122193102)